src/utilities/table/table.ts

```diff
@@ -159,3 +160,26 @@ export function stringFiltering(columnData: Array<string>, dataIndexes: number[]
     });
     return filteredDataIndexes;
 }
+
+export function hasFilterFunc(controlHeader: Map<string, ControlHeader>, queryResult: Map<number, ProcessedColumnData>): boolean {
+    const TRUE_REGEX = /^[tTyY].*$/;
```

Comment:
Rename it as `trueRegex`. Same for the next line.

Reply:
Please adjust to camelCase for consistency and perhaps remain the style for `AbstractCatalogProfileStore.TRUE_REGEX`.